master/docs/manual/configuration/global.rst

```diff
@@ -306,12 +306,11 @@ Three basic settings describe the buildmaster in status reports:
 
 :bb:cfg:`title` is a short string that will appear at the top of this buildbot installation's home page (linked to the :bb:cfg:`titleURL`).
 
-:bb:cfg:`titleURL` is a URL string that must end with a slash (``/``).
```

Comment:
I think it makes sense to keep this, otherwise users will all make an error of first trying with URL without a slash.

Reply:
Ah, the slash is added automatically, sorry.